document/cloud-client/analyze_form.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+def analyze_form(project_id, input_uri):
+    """Parse a form"""
+    # [START analyze_form]
+    from google.cloud import documentai_v1beta2 as documentai
+    from google.cloud.documentai_v1beta2 import types
```

Comment:
The region tag should be led by the product name, I put `documentai` but maybe it should be `document_ai` not sure which one you and Eric are working on. We also only want one region tag for the whole snippet. 

The newer style is to have imports at the top outside the function, we also typically prefer just to use one import and then access the `types` later. 
Oh on function variables we try to provide defaults now that the user can update
On `input_uri` if we have a publicly hosted form already that you can put that is sweet. I stole the one from the Node.js samples
```suggestion
# [START documentai_analyze_form]
from google.cloud import documentai_v1beta2 as documentai

def analyze_form(project_id='YOUR_PROJECT_ID', input_uri='gs://cloud-samples-data/documentai/invoice.pdf'):
    """Parse a form"""
```